extensions/wikia/Search/templates/WikiaSearch_exactWikiMatch.php

```diff
@@ -0,0 +1,30 @@
+<div class="exact-wiki-match">
+	<h1 class="exact-wiki-match__header"><?= wfMessage('wikiasearch3-related-wiki')->escaped(); ?></h1>
+	<div class="exact-wiki-match__result">
+		<a href="<?= $url ?>" title="<?= $title ?>" data-event="image">
+			<img src="<?= $imageURL ?>" alt="<?= $title ?>" class="exact-wiki-match__image" />
+		</a>
+		<div class="exact-wiki-match__content">
+			<h1 class="exact-wiki-match__wiki-header">
+				<a href="<?= $url ?>" data-event="title"><?= $title ?></a>
+			</h1>
+			<p class="exact-wiki-match__hub subtle"><?= $hub ?></p>
+			<ul class="exact-wiki-match__statistics">
+				<li>
+					<div><?= $pages['count'] ?></div>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "count" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)